tests/Functional/src/com/microsoft/aad/adal/test/WebRequestHandlerTests.java

```diff
@@ -162,26 +162,25 @@ public void testClientTraceInHeaders() {
 
     public void testNonExistentUrl() {
         WebRequestHandler request = new WebRequestHandler();
-        HttpWebResponse httpResponse = request.sendGet(
-                getUrl("http://www.somethingabcddnotexists.com"), null);
-        assertNotNull(httpResponse.getResponseException());
-        assertTrue("Unknown host exception",
-                httpResponse.getResponseException() instanceof UnknownHostException);
-        assertTrue("Unable to resolve host", httpResponse.getResponseException().getMessage()
-                .toLowerCase(Locale.US).contains("unable to resolve host"));
+        try {
+            request.sendGet(
+                    getUrl("http://www.somethingabcddnotexists.com"), null);
+        } catch (final IOException e) {
+            assertTrue(e instanceof UnknownHostException);
+            assertTrue(e.getMessage().toLowerCase(Locale.US).contains("unable to resolve host"));
+        }
     }
 
-    public void testGetWithIdRequest() {
+    public void testGetWithIdRequest() throws IOException {
         WebRequestHandler request = new WebRequestHandler();
         HttpWebResponse httpResponse = request.sendGet(getUrl(TEST_WEBAPI_URL + "/1"), null);
```

Comment:
All the tests related to this TEST_WEBAPI_URL is failing now since the webapi is no longer reachable. I'm not going to update those tests in this PR, will send a separate one to address them.